configs/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@aragon/osx-commons-configs",
+  "author": "Aragon Association",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.1",
```
We started with `0.0.1` everywhere.

---

done